blockware/fireworks/src/main.cpp

```diff
@@ -11,73 +11,163 @@
 #include <Particle.h>
 #include <DefaultConfig.h>
 
-#ifndef max
-#define max(X,Y) (((X) > (Y)) ? (X) : (Y))
-#endif
+#include <LIS2DW12Sensor.h>
+
 
 const int tlWidth = 40;
 const int tlHeight = 33;
-const float FIREWORK_CHANCE = 0.05;
-const int FIREWORKS_PER_FRAME = 64;
-const char *username = "@magusnn";
-const char *years = "5 years";
+const char *username = "@jcasts";
```

Comment:
Haha it's only fair that whoever made the last change here gets to leave their mark :)

Reply:
Oops, can change this to something more generic, what should I use?